domain-models-runtime/src/main/java/org/folio/rest/persist/PgExceptionFacade.java

```diff
@@ -18,13 +18,15 @@
    * @param throwable a GenericDatabaseException; any other Throwable is handled gracefully
    */
   public PgExceptionFacade(Throwable throwable) {
-    if (!(throwable instanceof GenericDatabaseException)) {
+    if (!(throwable instanceof PgException)) {
       fields = Collections.emptyMap();
       return;
     }
 
-    ErrorMessage errorMessage = ((GenericDatabaseException) throwable).getErrorMessage();
-    fields = errorMessage.getFields();
+    fields = new HashMap<>();
+    fields.put('M', ((PgException) throwable).getMessage());
+    fields.put('D', ((PgException) throwable).getDetail());
+    fields.put('C', ((PgException) throwable).getCode());
```

Comment:
getTable() and getIndex() can no longer return anything and should be removed.

Reply:
Oh yes.. Saw that the "hard way" in mod-inventory-storage, which in one place checked for 't'